### PR TITLE
Update instructions for manually running terratest tests

### PR DESCRIPTION
**Pull request by RyanStan**
As of Go v1.16, the environment variable GO111MODULE is set to on by default. Since the tests rely on GOPATH instead of modules, you have to set GO111MODULE to off.

I also made it explicit that to run the tests via Intellij, you have to have the Ultimate edition.

### CODE REVIEW

**Review by APegaDavis — APPROVED** [review]

**Review by MadhuriArugula — APPROVED** [review]

**Review by misterdorito — APPROVED** [review]

**Review by Saurabh-16 — APPROVED** [review]

**Comment by MadhuriArugula** [comment]
@taz-pega-work Can you please review the documentation changes?

File: terratest/run-tests-from-cmdline.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The test can also be run from the command line which allows you to avoid complyi
 - You have a local clone of this repo
 - You have installed an appropriate go version (v1.13 or later)
 - You have installed dep
+- You've disabled go modules by setting the environment variable `GO111MODULE="off"` (v1.16 or later)
```
Comment (taz-pega-work) [inline terratest/run-tests-from-cmdline.md:9]: keep spelling out the "you have" - why break the model on the fourth try?? ;-)
You have disabled go modules by setting the environment variable `GO111MODULE="off"` (v1.16 or later)

**Review by taz-pega-work — APPROVED** [review]
please consider one minor tweak

**Review by taz-pega-work — APPROVED** [review]
thank you!

**Pull request merged**